File: app/src/main/java/org/take365/Network/Models/Response/LoginResponse/LoginResult.java
package org.take365.Network.Models.Response.LoginResponse;

import java.io.Serializable;

/**
 * Created by evgeniy on 08.02.16.
 */
public class LoginResult implements Serializable {
    public int id;
    public String token;
    public String tokenExpires;
    public String username;
}

File: app/src/main/java/org/take365/Network/Models/Response/LoginResponse/LoginResult.kt
package org.take365.Network.Models.Response.LoginResponse

import java.io.Serializable

/**
 * Created by evgeniy on 08.02.16.
 */
data class LoginResult(
        var id: Int,
        var token: String,
        var tokenExpires: String?,
        var username: String
) : Serializable
